### PR TITLE
[SecurityInsight] Init release

### PR DESCRIPTION
**Pull request by bim-msft**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [x] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [x] Have you run `python scripts/ci/test_index.py -q` locally?

For new extensions:

- [x] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your PR is merged into master branch, a new PR will be created to update `src/index.json` automatically.  
The precondition is to put your code inside this repo and upgrade the version in the PR but do not modify `src/index.json`.

### CODE REVIEW

File: src/securityinsight/README.md [hunk 1]
```diff
@@ -0,0 +1,184 @@
+# Azure CLI sentinel Extension #
+This is the extension for sentinel
```
Comment (yungezz) [inline src/securityinsight/README.md:2]: why folder name not aligns with extension/command name?

File: src/securityinsight/README.md [hunk 2]
```diff
@@ -0,0 +1,184 @@
...
+Install this extension using the below CLI command
+```
+az extension add --name sentinel
+```
+
+### Included Features ###
+#### sentinel alert-rule ####
+##### Create #####
+```
+az sentinel alert-rule create --etag "\\"0300bf09-0000-0000-0000-5c37296e0000\\"" \
```
Comment (yungezz) [inline src/securityinsight/README.md:14]: pls mask subs id

File: src/securityinsight/README.md [hunk 3]
```diff
@@ -0,0 +1,184 @@
...
+    --data-connector-id "73e01a99-5cd7-4139-a149-9f2736ff2ab5" --resource-group "myRg" --workspace-name "myWorkspace" 
+```
+##### Show #####
+```
+az sentinel data-connector show --data-connector-id "763f9fa1-c2d3-4fa2-93e9-bccd4899aa12" --resource-group "myRg" \
+    --workspace-name "myWorkspace" 
+```
+##### Show #####
+```
+az sentinel data-connector show --data-connector-id "b96d014d-b5c2-4a01-9aba-a8058f629d42" --resource-group "myRg" \
```
Comment (yungezz) [inline src/securityinsight/README.md:101]: duplicate with above

File: src/securityinsight/README.md [hunk 4]
```diff
@@ -0,0 +1,184 @@
...
+az sentinel data-connector show --data-connector-id "07e42cb3-e658-4e90-801c-efa0f29d3d44" --resource-group "myRg" \
+    --workspace-name "myWorkspace" 
+```
+##### Show #####
+```
+az sentinel data-connector show --data-connector-id "c345bf40-8509-4ed2-b947-50cb773aaf04" --resource-group "myRg" \
+    --workspace-name "myWorkspace" 
+```
+##### Show #####
+```
```
Comment (yungezz) [inline src/securityinsight/README.md:130]: is this generated? lots of show examples same same syntax

File: src/securityinsight/README.md [hunk 5]
```diff
@@ -0,0 +1,184 @@
...
+    --workspace-name "myWorkspace" 
+```
+#### sentinel incident ####
+##### Create #####
+```
+az sentinel incident create --etag "\\"0300bf09-0000-0000-0000-5c37296e0000\\"" \
+    --description "This is a demo incident" --classification "FalsePositive" \
+    --classification-comment "Not a malicious activity" --classification-reason "IncorrectAlertLogic" \
+    --first-activity-time-utc "2019-01-01T13:00:30Z" --last-activity-time-utc "2019-01-01T13:05:30Z" \
+    --owner object-id="2046feea-040d-4a46-9e2b-91c2941bfa70" --severity "High" --status "Closed" --title "My incident" \
```
Comment (yungezz) [inline src/securityinsight/README.md:150]: mask object id

File: src/securityinsight/azext_sentinel/manual/__init__.py [hunk 6]
```diff
@@ -0,0 +1,12 @@
+# coding=utf-8
```
Comment (yungezz) [inline src/securityinsight/azext_sentinel/manual/__init__.py:1]: this module has no manual code?

File: src/securityinsight/README.md (hunk 1 above)
Comment (bim-msft) [reply src/securityinsight/README.md:2]: > why folder name not aligns with extension/command name?

@yungezz This was generated from Swagger definition.

File: src/securityinsight/README.md (hunk 4 above)
Comment (bim-msft) [reply src/securityinsight/README.md:130]: > is this generated? lots of show examples same same syntax

Yes it's auto generated, I will refine some.

File: src/securityinsight/azext_sentinel/manual/__init__.py (hunk 6 above)
Comment (bim-msft) [reply src/securityinsight/azext_sentinel/manual/__init__.py:1]: > this module has no manual code?

@yungezz Actually it has. I will split the manual code from the mixed generated code.

**Review by fengzhou-msft — APPROVED** [review]

**Pull request merged**